src/components/ChatEntry.js

```diff
@@ -1,22 +1,52 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+import { useState } from 'react';
 
 const ChatEntry = (props) => {
-  return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
-      <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
-      </section>
-    </div>
-  );
+	const [likePost, setLikePost] = useState(props.liked);
+
+	const toggleLike = () => {
+		const updatedEntry = {
+			id: props.id,
+			sender: props.sender,
+			body: props.body,
+			timeStamp: props.timeStamp,
+			liked: !props.liked,
+		};
+		setLikePost(!likePost);
```

Comment:
Looks like we are updating the chat message data from `entries` state using the `switchHeart`. Because of this, we aren't really _using_ `likePost` or displaying it in our component. 

Consider removing lines 8 and 18 completely